projects/api-twitter/server/utils/email/index.js

```diff
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+const sgTransport = require('nodemailer-sendgrid');
+const config = require('../../config');
+const { getHTML, replaceDoubleBraces } = require('./utils');
+
+const { sendGridKEY } = config;
+
+// eslint-disable-next-line object-curly-newline
+exports.mail = async ({ email, subject, template, data }) => {
+	const options = {
+		apiKey: sendGridKEY,
+	};
+
+	let htmlText = await getHTML(template);
+	htmlText = replaceDoubleBraces(htmlText, data);
+
+	const transporter = nodemailer.createTransport(sgTransport(options));
+
+	await transporter
+		.sendMail({
+			from: '"Juan Diego Carranza Vega" <jcarranzav1@hotmail.com>', // sender address
+			to: email,
+			subject,
+			text: htmlText,
+			html: htmlText,
+		})
+		.then(() => console.log('Correo enviado'))
+		.catch((e) => console.log(e));
```

Comment:
acá creo que seria mejor que usaran un `console.error(e)`

Reply:
y mismo comment de arriba usen una estructura en los console ej `[ERROR]: mail not sended whit error ${e}`